src/Helpers/ReportsHelper.js

```diff
@@ -38,7 +39,7 @@ export const buildFilters = filterData => {
             }
 
             case 'publish_date':
-                newValues[key].values = PUBLIC_DATE_OPTIONS.find(val => val.value === value).timespan_label;
+                newValues[key].values = PUBLIC_DATE_OPTIONS.find(val => val.value === value).label;
```

Comment:
:thinking: why?

Reply:
Because now the report header text is generated [here](https://github.com/RedHatInsights/vulnerability-ui/pull/1907/files#diff-cc08bdc0f31934740fc0927f66ff2e4b3f2f6746069dea06b4ff8bc8009e01bbR139) in a different more generic approach.